### PR TITLE
DISCO-143 Add environment variables to readme

### PR DESCRIPTION
**Pull request by jazairi**
#### What are the relevant tickets?
- https://mitlibraries.atlassian.net/browse/DISCO-143

#### Includes new or updated dependencies?
NO

### CODE REVIEW

**Review by matt-bernhardt — CHANGES_REQUESTED** [review]
We should also list the Sentry DSN - but I like the structure of this addition.

**Review by JPrevost — CHANGES_REQUESTED** [review]
Check JIRA for the bit about the PR template update as well we were going to do under this ticket.

**Review by JPrevost — APPROVED** [review]

File: README.md [hunk 1]
```diff
@@ -8,6 +8,15 @@ This repository will focus on the `DISCO UI` portion of this work.
...
 ![alt text](docs/diagrams/system_overview.png "Discovery system overview")
 
+## Environment variables
+
+### Required
+`VUE_APP_TIMDEX_API`: the root URL for the TIMDEX API
+
+### Optional
+`VUE_APP_RESULTS_PER_PAGE`: the number of records displayed per page
+`SENTRY_DSN`: set to Sentry project key to log exceptions
```
Comment (matt-bernhardt) [inline README.md:18]: Is this the right name? Or does it need to be prefixed with `VUE_APP_`?

Comment (jazairi) [reply README.md]: Thanks for catching this, @matt-bernhardt. Just pushed up the correction.

**Review by matt-bernhardt — APPROVED** [review]
Looks good - :shipit:

**Pull request merged**